web/src/components/DiffCard/CircularProgressWithLabel.jsx

```diff
@@ -13,17 +14,26 @@ const useStyles = makeStyles(theme => ({
     color: theme.palette.primary.light
   },
   top: {
-    color: theme.palette.primary.main,
     position: "absolute",
     left: 0
   },
+  badStatus: {
+    color: theme.palette.error.main
+  },
+  mediumStatus: {
+    color: theme.palette.warning.main
+  },
+  goodStatus: {
+    color: theme.palette.success.main
+  },
   circle: {
     strokeLinecap: "round"
   }
 }));
 export default function CircularProgressWithLabel(props) {
   const styles = useStyles();
-  const { value } = props;
+  let { value } = props;
+  value %= 100; // needed for demo

```

Comment:
I would not modify this directly like this. I would pass a flag in like, showColorBasedStatus in the prop. When it is set to true, then I perform this logic to show good, medium, or bad. Otherwise, this component can't be used for other purposes without the color logic.